tests/mocks/entities.ts

```diff
@@ -0,0 +1,17 @@
+import { BigInt } from '@graphprotocol/graph-ts'
+import { Nft } from '../../generated/schema'
+import { ZERO_ADDRESS } from '../../src/utils/constants'
+
+function createMockNft(tokenId: string): Nft {
+  const nft = new Nft('1')
```

Comment:
calling this function twice would create a conflict because nft with this id, would already been created. 
 
```suggestion
  const nft = new Nft(generateId(tokenId, ZERO_ADDRESS))
```